README.md

```diff
@@ -1,118 +1,44 @@
-# Barnardos consent form generator
+# consent-form-builder
 
 The purpose of the consent form generator is to allow a Barnardos researcher to
 enter details about an upcoming research session and produce consent forms that can
 initially be paper signed by a participant, and later be digitally signed.
 
-## Developer guide
+## Getting started
 
-The following sections of the readme outline the software stack used to create the application,
-how to get the application running and how to make and submit changes.
+Install:
```

Comment:
Do we think that someone new to a Ruby/Rails project would be able to set this up from this guide?